### PR TITLE
Add an invoice to a transaction that RPC returns

### PR DESCRIPTION
**Pull request by sgkim126**
It fixes https://github.com/CodeChain-io/codechain/issues/1287

### CODE REVIEW

**Comment by sgkim126** [comment]
I'll add test cases

File: rpc/src/v1/impls/chain.rs [hunk 1]
```diff
@@ -104,15 +104,23 @@ where
     }
 
     fn get_transaction(&self, transaction_hash: H256) -> Result<Option<Transaction>> {
-        Ok(self.client.transaction(&transaction_hash.into()).map(|tx| tx.into()))
+        let id = transaction_hash.into();
+        Ok(self.client.transaction(&id).map(|tx| {
+            let invoice = self.client.invoice(&id).expect("Invoice moust exist when transaction exists");
```
Comment (joojis) [inline rpc/src/v1/impls/chain.rs]: Typo 'moust'

File: rpc/src/v1/impls/chain.rs [hunk 2]
```diff
@@ -104,15 +104,23 @@ where
...
 
     fn get_invoice(&self, transaction_hash: H256) -> Result<Option<Invoice>> {
         Ok(self.client.invoice(&transaction_hash.into()))
     }
 
     fn get_transaction_by_tracker(&self, tracker: H256) -> Result<Option<Transaction>> {
-        Ok(self.client.transaction_by_tracker(&tracker).map(Into::into))
+        Ok(self.client.transaction_by_tracker(&tracker).map(|tx| {
+            let transaction_id = tx.hash().into();
+            let invoice = self.client.invoice(&transaction_id).expect("Invoice moust exist when transaction exists");
```
Comment (joojis) [inline rpc/src/v1/impls/chain.rs]: Same here

**Comment by sgkim126** [comment]
@joojis fixed

**Pull request merged**